### ISSUE TITLE
Loss too high

### ISSUE THREAD

**Issue by AICampB4** [sushant097/Handwritten-Line-Text-Recognition-using-Deep-Learning-with-Tensorflow#22]
Hello, Sir. Thank for your post. I'm a newbie to this domain and I read your post on Medium. I clone repos into Google Colab, download your data on kaggle and upload it to folder data as in readme. I also redefine images path in Data_Loader.py at line 75: 
fileName = filePath + 'self_lines' + fileNameSplit[0] + '.png'.
After training done: It has a really bad result. WER 400% and accuracy 0.00000%.
Could you help me figure out which part I have done wrong, Sir. Thank you in advance.

**Comment by Aditya-017** [comment]
Try to open it directly in gitpod :
u can open the project in gitpod by prefixing the the project link with gitpod.io/
like so gitpod.io/https://github.com/sushant097/Handwritten-Line-Text-Recognition-using-Deep-Learning-with-Tensorflow
then you can follow the instructions as given in README file. 
You can also use the pretrained model to get them read instructions in the README

**Comment by sushant097** [comment]
@AICampB4 your output looks strange. This means you setup incorrectly and not follow instructions in README. Following instructions in README will get rid of this error, and I assume you must be some level familiarity with TensorFlow and python to do that.

I am closing this issue, Please open another issue if you need.
